scripts/Notification_Reminder/notification.py

```diff
@@ -0,0 +1,11 @@
+import time
+from plyer import notification
+
+while True:
+    notification.notify(
+        title="Hey there!",
+        message="This is a pop-up reminder, take a break. Bye!",
+        # app_icon = "<path of the icon-image>",
+        timeout=10,
```

Comment:
Before showing the notification, ask the user what they want to be notified about and also take the time. The notification currently is hard coded, make it such that it is dynamic and changes based on what the user wants to be notified about. And finally make the timeout correspond to the time delay after which the user wants to be notified.